dynamic-plugins.default.yaml

```diff
@@ -479,6 +479,46 @@ plugins:
               accessKeyId: ${AWS_ACCESS_KEY_ID}
               secretAccessKey: ${AWS_SECRET_ACCESS_KEY}
 
+  - package: ./dynamic-plugins/dist/backstage-plugin-catalog-backend-module-bitbucket-cloud
+    disabled: true
+    pluginConfig:
+      # Uncomment the following to enable the Bitbucket Cloud catalog
+      # # https://backstage.io/docs/integrations/bitbucketCloud/discovery#configuration
+      # catalog:
+      #   providers:
+      #     bitbucketCloud:
+      #       yourProviderId: # identifies your ingested dataset
+      #         catalogPath: /catalog-info.yaml # default value
+      #         filters: # optional
+      #           projectKey: '^apis-.*$' # optional; RegExp
+      #           repoSlug: '^service-.*$' # optional; RegExp
+      #         schedule: # optional; same options as in TaskScheduleDefinition
+      #           # supports cron, ISO duration, "human duration" as used in code
+      #           frequency: { minutes: 30 }
+      #           # supports ISO duration, "human duration" as used in code
+      #           timeout: { minutes: 3 }
+      #         workspace: workspace-name
```

Comment:
Why is this commented out? Also the plugin is disabled by `disabled: true`, this config doesn't get applied unless the plugin is enabled and uncommenting this section doesn't enable the plugin.